specification/advisor/resource-manager/Microsoft.Advisor/stable/2020-01-01/advisor.json

```diff
@@ -1053,6 +1053,10 @@
           "additionalProperties": {
             "type": "string"
           }
+        },
+        "resourceMetadata": {
```

Comment:
How is this different than the 'metadata' field? These names will be confusing to customers in their current state.

Reply:
This object is to provide info for tracked resource on which recommendation is applied. Metadata is to provide info about recommendation and how to act upon it.

Does using resourceDetails remove confusion?